tests/queries/0_stateless/02352_grouby_shadows_arg.sql

```diff
@@ -0,0 +1,5 @@
+SET prefer_localhost_replica=0;
+SELECT toString(dummy) as dummy FROM remote('127.{1,1}', 'system.one') GROUP BY dummy; -- { echoOn }
```

Comment:
Sorry, I probably need to be more clear. Please add it once before queries:
```suggestion
-- { echoOn }
SELECT toString(dummy) as dummy FROM remote('127.{1,1}', 'system.one') GROUP BY dummy;
```
It's useful to get more readable `reference` files. And use `-- { echoOff }` when echoing is not necessary any more. Typically, it's test clean-up queries like`DROP TABLE`

Reply:
Thanks for explaining, didn't know this annotation existed. Updated the test now.